tools/confix/README.md

```diff
@@ -10,6 +10,50 @@ It is based on the [Tendermint RFC 019](https://github.com/tendermint/tendermint
 
 ## Installation
 
+### Add Config Command
+
+To add the confix tool, it's required to add the `ConfigCommand` to your application's root command file (e.g. `simd/cmd/root.go`).
+
+Import the `confixCmd` package:
+
+```go
+import "cosmossdk.io/tools/confix/cmd"
+```
+
+Find the following line:
+
+```go
+initRootCmd(rootCmd, encodingConfig)
+```
+
+After that line, add the following:
+
+```go
+rootCmd.AddCommand(
+    confixcmd.ConfigCommand(),
```

Comment:
if a users adds custom values to the app.toml how will the command know to add them when a migration happens? or does it generate a new config and compare

Reply:
Right now, it still only compares to the SDK default against a file, but in https://github.com/cosmos/cosmos-sdk/issues/14630 I'll improve the standalone functionalities. This is a good use case, as I was thinking of only SDK defaults, but comparing app default is probably better. Adding to the issue.